### PR TITLE
first pass of securitywiki helm chart

### PR DESCRIPTION
**Pull request by cmharlow**
Jira: https://mozilla-hub.atlassian.net/browse/SE-1153

What this PR does:
* creates a first pass of a securitywiki helm chart, with some pretty big overrides for CI to pass (biggest one of these overrides is that we can't use our securitywiki image to test the chart, since our image has Apache setup with OIDC, so the pod won't run without a valid OIDC connection - which I don't want to setup in CI).

### CODE REVIEW

**Review by duallain — APPROVED** [review]

File: charts/securitywiki/templates/hpa.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{{- if .Values.hpa.enabled }}
+apiVersion: autoscaling/v1
+kind: HorizontalPodAutoscaler
```
Comment (smarnach) [inline charts/securitywiki/templates/hpa.yaml:3]: Given that the wiki appears to be used by approximately 3 users, we probably don't strictly need this, but it won't hurt either.

**Review by smarnach — COMMENTED** [review]
This looks mostly like standard boilerplate, and I only have one question about the init container. I haven't tested this at all, but I see the Helm release in the stage cluster ran successfully, so it looks all good.

File: charts/securitywiki/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      maxSurge: 2
+      maxUnavailable: "25%"
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        {{- include "securitywiki.selectorLabels" . | nindent 8 }}
+    spec:
+      initContainers:
+        - name: migrate
```
[the remark this answers is not in the document]
Comment (cmharlow) [reply charts/securitywiki/templates/deployment.yaml:30]: discussed in person, will figure out an answer for this.

File: charts/securitywiki/templates/hpa.yaml (hunk 1 above)
Comment (cmharlow) [reply charts/securitywiki/templates/hpa.yaml:3]: probably should remove at some point.

**Pull request merged**